src/WorkflowManager/WorkflowManager/Controllers/WorkflowInstanceController.cs

```diff
@@ -179,18 +179,13 @@ public async Task<IActionResult> GetByIdAsync([FromRoute] string id)
         [Route("failed")]
         [HttpGet]
         [ProducesResponseType(typeof(IList<WorkflowInstance>), StatusCodes.Status200OK)]
-        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound)]
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
         public async Task<IActionResult> GetFailedAsync()
         {
             try
             {
                 var workflowInstances = await _workflowInstanceService.GetAllFailedAsync();
-                if (workflowInstances.IsNullOrEmpty())

```

Comment:
is it possible to be null is it ok to just return null in that scenario? or do we need to specify  if its null return empty list?

Reply:
It will return an empty list if there are none here.

Reply:
as in it wil never be null